### PR TITLE
更新对应于FISCO-BCOS 3.0的第一、二章内容

### PR DESCRIPTION
**Pull request by shuo-young**

### CODE REVIEW

**Comment by shuo-young** [comment]
section 1&2 modified by shuoyang

File: docs/quickstart/prerequisite.md [hunk 1]
```diff
@@ -46,7 +46,14 @@ rustup component add rustc-dev
...
 
-   由于Liquid使用了少量目前尚不稳定的Rust语言特性，因此在构建时需要依赖nightly版本的 ``rustc`` 。但是这些特性目前已经被广泛应用在Rust项目中，因此其可靠性值得信赖。随着Rust语言迭代演进，这些特性终将变为稳定特性。
+   由于Liquid使用了少量目前尚不稳定的Rust语言特性，因此在构建时需要依赖 ``nightly`` 版本的 ``rustc`` 。但是这些特性目前已经被广泛应用在Rust项目中，因此其可靠性值得信赖。随着Rust语言迭代演进，这些特性终将变为稳定特性。
+```
+
+查看当前rustup已安装的版本并切换为nightly
+
+```shell
+rustup toolchain list
+rustup default nightly
```
Comment (bxq2011hust) [inline docs/quickstart/prerequisite.md]:
```suggestion
rustup default nightly-2021-06-23
```

File: docs/quickstart/prerequisite.md [hunk 2]
```diff
@@ -83,15 +90,23 @@ registry = "git://mirrors.ustc.edu.cn/crates.io-index"
...
+.. admonition:: 注意
+
+   若无法正常访问GitHub，则请执行 ``cargo install --git https://gitee.com/WeBankBlockchain/cargo-liquid --branch dev --force`` 命令进行安装。
 ```
 
 ```eval_rst
 .. admonition:: 注意
 
-   若无法正常访问GitHub，则请执行 ``cargo install --git https://gitee.com/WeBankBlockchain/cargo-liquid --tag v1.0.0-rc1 --force`` 命令进行安装。
+   请确保配置 ``cmake`` 环境，Linux环境下建议手动安装cmake，建议安装最新版本。
```
Comment (bxq2011hust) [inline docs/quickstart/prerequisite.md]: 这个cmake是不是不需要安装？

File: docs/quickstart/prerequisite.md (hunk 1 above)
Comment (shuo-young) [reply docs/quickstart/prerequisite.md]: 这个版本会改的吧

File: docs/quickstart/prerequisite.md (hunk 2 above)
Comment (shuo-young) [reply docs/quickstart/prerequisite.md]: 我在空docker上提示要安装，这个我再确认一下

Comment (shuo-young) [reply docs/quickstart/prerequisite.md]: > 

running: "cmake" "/root/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/wabt-sys-0.8.0/wabt" "-DBUILD_TESTS=OFF" "-DBUILD_TOOLS=OFF" "-DCMAKE_INSTALL_PREFIX=/tmp/cargo-installLkzhbX/release/build/wabt-sys-330d517735be39df/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC" "-DCMAKE_CXX_COMPILER=c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=Release"
  -- The C compiler identification is GNU 9.3.0
  -- The CXX compiler identification is unknown
  -- Check for working C compiler: /usr/bin/cc
  -- Check for working C compiler: /usr/bin/cc -- works
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Configuring incomplete, errors occurred!
  See also "/tmp/cargo-installLkzhbX/release/build/wabt-sys-330d517735be39df/out/build/CMakeFiles/CMakeOutput.log".
  See also "/tmp/cargo-installLkzhbX/release/build/wabt-sys-330d517735be39df/out/build/CMakeFiles/CMakeError.log".

**Comment by shuo-young** [comment]
section 1&2 modified by ys 1202 reviewed suggestions

**Comment by shuo-young** [comment]
section 1&2 modified by ys 1202

**Review by bxq2011hust — APPROVED** [review]

**Pull request merged**